src/main/java/org/sagebionetworks/web/client/widget/discussion/ReplyWidget.java

```diff
@@ -1,26 +1,39 @@
 package org.sagebionetworks.web.client.widget.discussion;
 
 import org.sagebionetworks.repo.model.discussion.DiscussionReplyBundle;
+import org.sagebionetworks.web.client.GWTWrapper;
+import org.sagebionetworks.web.client.PortalGinInjector;
+import org.sagebionetworks.web.client.widget.user.UserBadge;
 
+import com.google.gwt.inject.client.Ginjector;
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
 
 public class ReplyWidget implements ReplyWidgetView.Presenter{
 
 	ReplyWidgetView view;
+	GWTWrapper gwtWrapper;
+	PortalGinInjector ginInjector;
 
 	@Inject
 	public ReplyWidget(
-			ReplyWidgetView view
+			ReplyWidgetView view,
+			GWTWrapper gwtWrapper,
+			PortalGinInjector ginInjector
```

Comment:
Same, just inject UserBadge once in constructor (and add widget to the view).  Then configure the UserBadge in configure()